examples/cryptorealtime/terraform-setup/service_account.tf

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
This should probably be 2019 since this is a new file being checked in, unless you wrote it in 2018.